Documentation/operations/upgrade.rst

```diff
@@ -521,15 +820,15 @@ IMPORTANT: Changes required before upgrading to 1.8.0
   all nodes to participate. In order to ensure scalability, ``CiliumNetworkPolicy``
   status visibility has been disabled for all new deployments. If you want to
   enable it, set the ConfigMap option ``disable-cnp-status-updates`` to false or
-  set the Helm variable ``--set config.enableCnpStatusUpdates=true``.
+  set the Helm variable ``--set enableCnpStatusUpdates=true``.
 
 * Prior to 1.8 release, Cilium's eBPF-based kube-proxy replacement was not able
   to handle Kubernetes HostPort feature and therefore CNI chaining with the
-  ``portmap`` plugin (``global.cni.chainingMode=portmap``) was necessary while
-  turning off the kube-proxy replacement (``global.kubeProxyReplacement=disabled``).
+  ``portmap`` plugin (``cni.chainingMode=portmap``) was necessary while
+  turning off the kube-proxy replacement (``kubeProxyReplacement=disabled``).
   Starting from 1.8, CNI chaining is no longer necessary, meaning Cilium can be
   used natively to handle HostPort when running with Cilium's kube-proxy replacement.
-  That is, for ``global.kubeProxyReplacement=probe`` and ``global.kubeProxyReplacement=strict``
+  That is, for ``kubeProxyReplacement=probe`` and ``kubeProxyReplacement=strict``
```

Comment:
This section is the v1.8 upgrade notes. Why are we changing the Helm options here?

Reply:
fixed